### PR TITLE
Add overlay for Huawei MAR

### PR DESCRIPTION
**Pull request by yihanwu1024**

### CODE REVIEW

**Comment by yihanwu1024** [comment]
I built successfully on my own machine though. There may be a few extra attributes.

**Comment by yihanwu1024** [comment]
Problem resolved.

File: Huawei/kirin710/MAR/res/xml/power_profile.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <value>400000</value>
+    </array>
+    <array name="cpu.active.cluster0">
+        <value>0.1</value>
+    </array>
+    <item name="cpu.idle">0.1</item>
+    <array name="memory.bandwidths">
+        <value>22.7</value>
+    </array>
+    <item name="battery.capacity">1000</item>
```
Comment (phhusson) [inline Huawei/kirin710/MAR/res/xml/power_profile.xml]: x Doubt

Comment (yihanwu1024) [reply Huawei/kirin710/MAR/res/xml/power_profile.xml]: I looked at all images and found three overlays, but only one apk contains power_profile. May need more time to dig around, but other files are okay. If you feel comfortable you can merge the other files for now.

Comment (phhusson) [reply Huawei/kirin710/MAR/res/xml/power_profile.xml]: on huawei devices power_profile is not in an apk. just look for files called power_profilesomething.xml

**Pull request closed without merging**